eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -1,6 +1,7 @@
 steps:
   - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
     - pwsh: |
+        $PrSourceBranch = "$(System.PullRequest.SourceBranch)"
```

Comment:
Lets just eliminate this variable as it isn't really needed. I gave a suggest below.